bin/templates/dropdown/template.tmpl

```diff
@@ -1,8 +1,8 @@
 <div class="share ns__share-dropdown">
-    <div class="share__button">
+    <button class="share__button">
         <div class="share__png_icon"></div>
         <p><%= label %></p>
-    </div>
+    </button>
```

Comment:
Hi Chris. I totally missed your comments.

I'm not sure about removing the `<p>` tags. Maybe it should be a header rather than a paragraph but having that text block in a distinct element seems useful.

It looks like someone has already changed the Share `div` to a `button` for the Issues Guide and it looks fine in my Chrome at least. However, changing the Close element to a `button` might need a little CSS tweaking.